libs/coin-evm/src/__tests__/unit/adapters/etherscan.unit.test.ts

```diff
@@ -19,6 +19,63 @@ describe("EVM Family", () => {
   describe("adapters", () => {
     describe("etherscan", () => {
       describe("etherscanOperationToOperations", () => {
+        it("should convert an etherscan-like smart contract creation operation (from their API) to a Ledger Live Operation", () => {
```

Comment:
Could be interesting to test as well the case where the address is badly formatted **but** does not fall in this condition: `if (!addr || addr === "0x" || addr === "0x0")`

Reply:
https://github.com/LedgerHQ/ledger-live/pull/5546/commits/15f50b21c5fb2b77a1fa692d4158ab5951f05b9b#diff-47974acb4fd1dfce54213ad9fc7cef06165b4e2463628370e022528b8223547d